CONTRIBUTING.md

```diff
@@ -111,3 +115,63 @@ A PR can only be merged into master by a maintainer, if all of these conditions
 * It has been approved by at least two maintainers. If it was a maintainer who opened the PR, only one extra approval is needed.
 * It has no requested changes.
 * It is up to date with current master.
+
+### Release process
+
+Maintainers need to perform the following actions **in the order described here** to push out a release.
+
+#### Verify the version in [DESCRIPTION](./DESCRIPTION) file
+
+Create a new PR and merge it, if the version in the [DESCRIPTION](./DESCRIPTION) file is not good.
+
+#### GitHub issues and milestones
+
+**Note:** we always put issues related to a version in a Milestone whose name matches the version.
+
+- Ensure the name of the milestone used for the new release version matches the name of the tag/version that has just been pushed. Renamed it if needed.
+- Clean this opened milestone if some issues are still opened (move them to a new one or discard milestone from them)
+- Close the milestone
```

Comment:
⚠️ we are not using this way of doing in this repository, so I suggest we remove this paragraph